spec-dtslint/Observable-spec.ts

```diff
@@ -1,6 +1,22 @@
 import { Observable, of, OperatorFunction } from 'rxjs';
 import { mapTo } from 'rxjs/operators';
 
+// The `a` (for argument) function is used to keep the tests uncluttered.
```

Comment:
If you do a `/** Info here */` style doc-comment, IDEs like VS Code will pick them up. :)

Reply:
Done.